client/components/reviewWidget/breakdownComponents/ReviewsSummary.jsx

```diff
@@ -1,16 +1,28 @@
 import React from 'react';
 import StarAverage from '../../shared/StarAverage.jsx';
 
-const ReviewsSummary = () => {
+const calcPercentRecommended = (recommended) => {
+  let recomend = parseInt(recommended.true);
+  let total = recomend + parseInt(recommended.false);
+  return recomend / total * 100 + '%';
+};
+
+const ReviewsSummary = ({reviewAverage, recommended, reviewCount}) => {
```

Comment:
chrome devtools says reviewCount is undefined, probably related to issue I found earlier.